clmm/theory/func_layer.py

```diff
@@ -465,7 +465,9 @@ def compute_reduced_tangential_shear(
                 `r_proj` is a float);
             * `applegate14`: use the equation (6) in Weighing the Giants - III \
                 (Applegate et al. 2014; https://arxiv.org/abs/1208.0605) to evaluate tangential reduced shear;
-
+            * `schrabback`: use the equation (12) in Cluster Mass Calibration at High Redshift \
```

Comment:
Do you mind using "schrabback18" as the option name (add year)? To be consistent with the other one. Thanks.